castervoice/lib/automation.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+import io
+import os
+# import re
+import sys
+import traceback
+from __builtin__ import True
+from subprocess import Popen
+import toml
+import time
+
+from castervoice.lib.actions import Key, Text
+from castervoice.lib.context import AppContext, read_selected_without_altering_clipboard
+from castervoice.lib.utilities import load_toml_file
+from castervoice.lib import settings
+
+_USER_DIR = os.path.expanduser("~").replace("\\", "/") + "/.caster"
+
+def _rebuild_items(config):
+    # logger.debug('Bring me rebuilding extras')
+    return {
+        key: (os.path.expandvars(value), header)
+        for header, section in config.iteritems() for key, value in section.iteritems()
+    }
+
+def github_branch_pull_request():
+    # Function to fetch a PR and make a new branch
+    try:
+        import natlink
+        context = AppContext(executable="chrome") | AppContext(executable="iexplore") | AppContext(executable="firefox")
+        if context:
+            print("Checking out pull request locally.")
+            Key("c-l/20").execute()
+            print("Checking out pull request locally 2.")
+            url = read_selected_without_altering_clipboard()
+            print(url[1])
+            if url[0] == 0:
+                split_string = url[1].split("/pull/")
```

Comment:
Can this be named better

Reply:
Which name?

Reply:
Yes. Maybe split the array into two variables with names